src/platform/site-wide/user-nav/components/Typeahead.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import environment from 'platform/utilities/environment';
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import debounce from 'platform/utilities/data/debounce';
 
-export const isTypeaheadEnabled =
-  !environment.isProduction() && document.location.pathname === '/';
```

Comment:
Oh interesting, so this means that now it'll now show up on every page when it's enabled

Reply:
That's fine IMO now that Flipper is being implemented

Reply:
Ahh good catch! I will make sure that is something we want before this gets merged in!